prac_05/word_occurrences.py

```diff
@@ -20,7 +20,9 @@ def main():
 
```

Comment:
You also need to add an input code, so the user can input text. text = input("text:  ").